elo_boat.py

```diff
@@ -34,11 +34,11 @@
 client = discord.ext.commands.Bot(command_prefix='?', case_insensitive=True, intents=intents)
 client.remove_command("help")
 
-GUILD_NAME = "Battleships"  # "BotTest" #
+GUILD_NAME = "Battleships"
 _guild = None
 baseurl = "https://api.wc3stats.com"
-leaderboard_channel_id = 716379545095241809  # test channel 804443429957664808 #716379545095241809
-upload_channel_id = 646155910640697375  # test channel 804443389641752597 #646155910640697375
+leaderboard_channel_id = 716379545095241809
+upload_channel_id = 646155910640697375
 upload_channel = None
 admin_role_id = 461111074523971584  # test channel 698548028591570994 // bscf 461111074523971584

```

Comment:
i don't rly know why you want to remove the useful testing info, but if you remove, i guess you should remove all of them ^^

Reply:
Will probably remove more later